### PR TITLE
Use python3.8 and add logging

### PR DESCRIPTION
**Pull request by bityob**
- [x] Use python latest docker image (Python 3.8)
- [x] Add logging 
- [x] Ignore certificates errors
- [ ] Set ignore certificates based on configuration (default will be to not ignore such errors)

Verification locally -

image

image

### CODE REVIEW

File: dockerbot.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import sys
```
Comment (bityob) [inline dockerbot.py]: remove `import sys`

**Comment by AdamRussak** [comment]
@bityob Great Loggin! like it 👍 

but please remove handling output to Telegram, we wish to keep it clean.
you can change the Success message to "Signed" if you wish.

please fix and we will approve the PR

**Comment by AdamRussak** [comment]
PS 
congrats for being our first PR :)

**Comment by bityob** [comment]
@AdamRussak 

Created new PR with the fix #2 

Thanks 👍

**Pull request closed without merging**